docs/CHANGELOG.md

```diff
@@ -16,6 +16,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+#### TOML Configuration (optional)
+
+Chainlink now supports static configuration via TOML files as an alternative to the existing combination of environment variables and persisted database configurations.
+This is currently _optional_, but in the future (with `v2.0.0`), it will become *mandatory* as the only supported configuration method.
```

Comment:
```suggestion
This is currently _optional_ and _experimental_, but in the future (with `v2.0.0`), it will become *mandatory* as the only supported configuration method.
```

Reply:
It actually shouldn't be experimental any longer at this point - at least not that particular term.